mithril-common/src/entities/cardano_transaction.rs

```diff
@@ -1,13 +1,12 @@
 use crate::crypto_helper::MKTreeNode;
 
-use super::{BlockNumber, ImmutableFileNumber};
+use super::{
+    cardano_chain_point::{BlockHash, SlotNumber},
+    BlockNumber, ImmutableFileNumber,
+};
```

Comment:
We should avoid `use super::` in production code, (it's fine for tests). can you replace it with a `use crate::entities::` instead ?

Reply:
Done